internal/parse/parse.go

```diff
@@ -101,10 +102,14 @@ func GetBindMount(ctx *types.SystemContext, args []string, contextDir string, st
 			if len(kv) == 1 {
 				return newMount, "", fmt.Errorf("%v: %w", kv[0], errBadOptionArg)
 			}
-			if err := parse.ValidateVolumeCtrDir(kv[1]); err != nil {
+			targetPath := kv[1]
+			if !path.IsAbs(targetPath) {
+				targetPath = filepath.Join(workDir, filepath.Clean(targetPath))
```

Comment:
I was thinking

				targetPath = filepath.Clean(filepath.Join(workDir, targetPath))

Reply:
In otherplaces in buildah we first clean the user input and then join it with a predefined path. I am just wondering if there any edge-case we miss cleaning the path later.

Reply:
@rhatdan I was looking for edge-case so I think as @nalind suggested and doc say I think `filepath.Join(` already performs a `clean(` https://pkg.go.dev/path/filepath#Join

Reply:
Ok, do we care about eval symlinks?

Reply:
@rhatdan Not really since this expects to be always executed on host.